code/data_structures/stack/balanced_expression/balanced_expression.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+
+class balanced_expression {
+  public static void main(String args[]) {
+		System.out.println("Balanced Expression");
+		Scanner in = new Scanner(System.in);
+		String input = in.next();
+		boolean output = is_expression_balanced(input);
+		if (output == true) System.out.println("The expression is balanced");
+		else System.out.println("The expression is not balanced");
+  }
+
+  static boolean is_expression_balanced(String input) {
```

Comment:
Method name violates Java naming convention. It should be isExpressionBalanced instead.